source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,185 @@
+
+```jsx
+// Customizing retries for an individual test
+describe('User sign-up and login', () => {
+  // `it` test block with no custom configuration
+  it('should redirect unauthenticated user to sign-in page', () => {
+    ...
+  })
+
+  // `it` test block with custom configuration
+  it('allows user to login', { 
+    retries: {
+      runMode: 3,
+      openMode: 2
+    }
+  }, () => {
+    ...
+  })
+})
+
+```
+
+#### Test Suite(s)
+
+If you want a suite of tests to re-run in requires a custom number of test retries configured, this can be achieved by using the test's local configuration option
+
+```jsx
+// Customizing retries for a suite of tests
+describe('User bank accounts', {
+  retries: {
+    runMode: 3,
+    openMode: 1,
+  }
+}, () => {
+  // Individual tests will be run normally
+  it('allows a user to view their transactions, () => {
+    ...
+  }
+
+  it('allows a user to edit their transactions, () => {
+    ...
+  }
+})
+
+```
+
+You can find more information about custom configurations here: {% url "Test Configuration" configuration#Test-Configuration %}
+
+## Assets
+
+With test retries, Cypress will now generate any assets (i.e., screenshots, video recordings, etc.) per each test attempt. The assets will be properly labeled with the attempt number appended at the end of each asset.
+
+    🖼 Insert screenshot
+
+## Tips and Strategies
+
+While test retries are great for helping to avoid false negatives from failing an entire test run, it is not a good replacement for writing good tests. As a result, here are some tips and strategies to keep in mind in order to maximize the effectiveness of your tests with test retries:
+
+- If you are noticing that you need to increase the number of retries, this cause is more likely due to how the tests are written and is worth spending the time to investigate
+- If you use `Run all specs` a lot in `cypress open` mode, make sure to configure your Cypress instance to have a global `retries` of `2` so you can simulate what is being run in `cypress run` mode.
+
+## Frequently Asked Questions (FAQs)
+
+#### Will runs with test retries be counted as more than one run?
+
+No. When recording to the Cypress Dashboard, runs with test retries will be counted as a single run.
+
+
+{% note warning 'Firefox Garbage Collection' %}
+Cypress triggers Firefox's internal garbage collection (GC) to better manage the browser's memory consumption. {% url "Learn more here" configuration#firefoxGcInterval %}.
+{% endnote %}
+
+Excluding {% url "Electron" launching-browsers#Electron-Browser %}, any browser you want to run Cypress tests in needs to be installed on your local system or CI environment. A full list of detected browsers is displayed within the browser selection menu of the {% url "Test Runner" test-runner %}.
+
+{% imgTag /img/guides/cross-browser-testing/cypress-browser-selector.png "Cypress Test Runner with Firefox selected as the browser" "no-border" %}
```

Comment:
I don't understand why this is here. 🤔 What does Firefox GC have to do with test retries? What does needing to install browsers have to do with test retries? Seems out of place.

If necessary, this is a better image of the browser dropdown imo, since it highlight all browsers we support. I'm not sure how we ended up with 2 images. 

{% imgTag /img/guides/browser-list-dropdown.png "Select a different browser" %}

Reply:
Ah these were left for snippet reference purposes. They will be deleted shortly once I add the new screenshots